files/grest/rpc/assets/asset_info.sql

```diff
@@ -59,27 +79,22 @@ BEGIN
         AND 
         ARC.asset_name = _asset_name
       LIMIT 1
-    ) AS token_registry_metadata,
-    (
-      SELECT
-        SUM(MTM.quantity)::text AS amount
-      FROM
-        ma_tx_mint MTM
-      WHERE
-        MTM.ident = _asset_id
-    ) AS total_supply,
-    (
+    ) AS token_registry_metadata
+  FROM 
+    multi_asset MA
+    LEFT JOIN LATERAL (
       SELECT
-        MIN(B.time) AS date
+        MIN(B.time) AS date,
+        SUM(MTM.quantity)::text AS amount,
```

Comment:
should we call this total supply?

Reply:
It is in the final output, but if you want to rename this internal column as well I'm fine with that.